doc/build/orm/join_conditions.rst

```diff
@@ -700,7 +700,7 @@ directly.  A query from ``A`` to ``D`` looks like:
 
 .. sourcecode:: python+sql
 
-    sess.query(A).join(A.d).all()
+    select(A).join(A.d))
```

Comment:
should probably be `sess.scalars(...).all()`

Reply:
I wasn't sure if this was intentional or not, but if you look at the following lines there's a representation of the generated SQL, which looks to me as if this could be the result of running the above statement in a REPL session, so not executing the statement would make sense here, since that's what would create the output shown below. 

But maybe - if my assumption is correct - this would be something more appropriate for a different PR, as to not mix up two objectives here?

Reply:
I'm not actually sure here. It may be that the sql is intended to be the log? I guess we will go with what @zzzeek prefers

Reply:
when you see that {opensql} or {sql} thing, that is meant to illustrate the engine logging SQL going to the database. so above, the statement would have to be executed.

in the tutorials, which are tested w/ doctest, this all has to be exactly correct for them to pass.  in other docs, things are clearly not that consistent, but we should probably keep the convention that if there is Python code, followed by {sql} or {opensql}, it means, "the statement was executed".  

here that means the equilvalent statement to 

    sess.query(A).join(A.d).all()

is

    sess.scalars(select(A).join(A.d))).all()

Reply:
Thanks for the clarification! I'll make the appropriate changes